examples/src/main/python/ml/count_vectorizer_example.py

```diff
@@ -17,23 +17,26 @@
 
 from __future__ import print_function
 
-from pyspark.sql import SparkSession
 # $example on$
 from pyspark.ml.feature import CountVectorizer
 # $example off$
+from pyspark.sql import SparkSession
+
 
 if __name__ == "__main__":
-    spark = SparkSession\
-        .builder\
-        .appName("CountVectorizerExample")\
-        .getOrCreate()
+    spark = (SparkSession
+             .builder
+             .appName("CountVectorizerExample")
+             .getOrCreate())
 
     # $example on$
     # Input data: Each row is a bag of words with a ID.
-    df = spark.createDataFrame([
-        (0, "a b c".split(" ")),
-        (1, "a b b c a".split(" "))
-    ], ["id", "words"])
+    df = spark.createDataFrame(
+        [
+            (0, "a b c".split(" ")),
+            (1, "a b b c a".split(" "))
+        ],
```

Comment:
Indeed, this is a recommendation not an obligation. I see it to be more looking like Scala multi-line code, and I prefer it. It is a personal opinion, and I don't think there is a pylint/pep8 check to prevent using \.